site/content/en/docs/Getting Started/create-fleet.md

```diff
@@ -339,7 +339,7 @@ allocated a `GameServer` out of the fleet, and you can now connect your players
 A handy trick for checking to see how many `GameServers` you have `Allocated` vs `Ready`, run the following:
 
 ```
-kubectl get gs -o=custom-columns=NAME:.metadata.name,STATUS:.status.state,IP:.status.address,PORT:.status.ports
+kubectl get gs
```

Comment:
outputs here are also still old for fleets:
https://107091d-dot-preview-dot-agones-images.appspot.com/site/docs/getting-started/create-fleet/